
\documentclass[letterpaper,portrait,11pt]{scrartcl}

\usepackage[T1,T3]{fontenc}
\usepackage{fourier}
\usepackage[english]{babel}															% English language/hyphenation
\usepackage[protrusion=true,expansion=true]{microtype}				% Better typography
\usepackage{fullpage}
\usepackage{sectsty}
\usepackage[toc,page]{appendix}

%% test addition 2

% linking options
\usepackage[pdftex]{graphicx}														% Enable pdflatex
\usepackage[pdftex,
  colorlinks=true,
  urlcolor=blue,
  filecolor=blue,
  linkcolor=blue,
  citecolor=blue,
  pdftitle={St Anns Bank Framework Assessment},
  pdfauthor={Jae S. Choi, Angelia S.M. Vanderlaan, Gordana Lazin, Mike McMahon,  Ben Zisserson, Brent Cameron, Jenna Munden},
  pdfkeywords={research document scotian shelf marine protected area assessment},
  pdfpagemode=None,
  bookmarksopen=true]{hyperref}

%% biblio
%\usepackage[backend=biber,style=alphabetic,sorting=ynt]{biblatex}
%\addbibresource{sab.bib}  %% the bibtex file
%\usepackage{natbib}
%\bibpunct[, ]{(}{)}{;}{a}{}{,}
% \citeindextrue
% see  /usr/share/texmf/tex/latex/natbib/natbib.sty for more codes
% natbib examples
%\citet{goossens93}     Goossens et al. (1993)
%\citep{goossens93}     (Goossens et al., 1993)
%\citet*{goossens93}    Goossens, Mittlebach, and Samarin (1993)
%\citep*{goossens93}    (Goossens, Mittlebach, and Samarin, 1993)
%\citeyear, \citeauthor
%\citealt


%%% Equation and float numbering
\usepackage{amsmath,amsfonts,amsthm}										% Math packages
\numberwithin{equation}{section}		% Equationnumbering: section.eq#
\numberwithin{figure}{section}			% Figurenumbering: section.fig#
\numberwithin{table}{section}				% Tablenumbering: section.tab#

%%% Maketitle metadata
\newcommand{\horrule}[1]{\rule{\linewidth}{#1}} 	% Horizontal rule

\title{
		%\vspace{-1in} 	
		\usefont{OT1}{bch}{b}{n}
		\horrule{0.5pt} \\[0.4cm]
		\huge Assessment Framework for the St Anns Bank Marine Protected Area  \\
		\horrule{2pt} \\[1.0cm]
}
\author{
		\normalfont \normalsize  \textsc{Jae S. Choi, Angelia S.M. Vanderlaan, Gordana Lazin,} \\
    \normalfont \normalsize  \textsc{Mike McMahon,  Ben Zisserson, Brent Cameron, Jenna Munden} \\ [25pt]		
		\normalfont \normalsize \textsc{Population Ecology Division} \\ 
    \normalfont \normalsize \textsc{Fisheries and Oceans Canada} \\ 
    \normalfont \normalsize \textsc{Bedford Institute of Oceanography} \\ [25pt]
    \normalsize \textsc{\today}
}

\date{}
%%% misc commands
\newcommand{\ecomod}{\string~/ecomod_data/}   %  \string~ is a representation of the home directory 
\newcommand{\mpa}{\ecomod/mpa/}       %  \string~ is a representation of the home directory 
\newcommand{\sab}{\ecomod/mpa/sab/}   %  \string~ is a representation of the home directory 
\newcommand{\analysis}{\ecomod/mpa/analysis/}   %  \string~ is a representation of the 


\begin{document}

\maketitle

\setlength{\parskip}{12 pt} % spacing between paragraphs
\setlength{\parindent}{0cm}
\setlength{\floatsep}{2cm}
\setcounter{tocdepth}{2} % <-- 2 includes up to subsections in the ToC
\setcounter{secnumdepth}{3} % <-- 3 numbers up to subsubsections
\tableofcontents 

\section*{Acknowledgements}

We highlight the invaluable assistance of a great number of scientists that have been part of the various surveys that inform this document. This report could not have been completed without their guidance and assistance: DataShop (Shelley Bond), Remote Sensing Group (Carla Caverhill, George White), AZMP (Catherine Johnson, Benoit Casault), Snow crab survey (Ben Zisserson, Brent Cameron, snow crab industry), Groundfish survey (Scott Wilson, Bill MacEachern, Don Clark), Clam survey (Dale Roddick), substrate grain size (Vladimir Kostelev, Charles Hannah), bathymetry data (Canadian Hydrographic Service), ocean temperature data (Roger Pettipas (Ocean Sciences).

\section*{Abstract}
\clearpage

\section{Introduction}

\subsection{Terms of reference}
The Health of the Oceans (HOTO, 2007-2012) and the National Conservation Plan (NCP, 2014-2019) initiatives support the conservation and restoration of lands and waters in Canada. In this context, DFO Science has been tasked with developing a monitoring approach for Marine Protected Areas (MPAs) and if possible to assess their effectiveness in meeting their objectives. 

\subsection{Scope of this report}
An MPA is defined in the Oceans Act 35(1) as, "An area of the sea that forms part of the internal waters of Canada, the territorial sea of Canada or the exclusive economic zone of Canada and has been designated under this section for special protection for one or more of the following reasons: 

\begin{itemize}
  \item The conservation and protection of commercial and non-commercial fishery resources, including marine mammals, and their habitats; 
  \item The conservation and protection of endangered or threatened marine species, and their habitats; 
  \item The conservation and protection of unique habitats; 
  \item The conservation and protection of marine areas of high biodiversity or biological productivity; and; 
  \item The conservation and protection of any other marine resource or habitat as is necessary to fulfill the mandate of the Minister."
\end{itemize}

As a result, for the purposes of this report, we will likewise focus upon these key ecosystem attributes or components, namely: \textbf{productivity, biodiversity, habitat, and  species of interest}. In reality, however, there are many other attributes or components of ecosystems that are also known to be important and relevant. They include: ecological integrity and health, trophic structure and balance, ecosystem function, complexity, network structure, resilience, sustainability as well an open-ended number of species or life history stages of the resident species. These other components will be touched upon where possible or necessary but will receive less attention as the legislated mandate is quite clear. 

\subsection{St. Anns Bank}
The St. Anns Bank Marine Protected Area (herein, SAB) is an area of interest for eventual designation as an MPA. It is located east of Cape Breton Island, Nova Scotia, Canada (Figure~\ref{figSABCloseup}). An advisory process (DFO 2012), identified the primary objectives of SAB as being to conserve, protect and where appropriate restore, ecologically distinctive or significant areas, and overall, the ecosystem "health" of SAB. As in the Oceans Act, the focus was upon the above four ecosystem components: \textbf{productivity, biodiversity, habitat and species of interest}.

Other MPA goals were also expressed in DFO (2012), but these were less emphatic:

\begin{itemize}
	\item Contribute to the health, resilience and restoration of the Eastern Scotian Shelf ecosystem
	\item Contribute to the recovery and sustainability of commercial fisheries 
	\item Promote scientific research and monitoring to further understand and protect SAB
\end{itemize}

In, An Ecological Overview and Assessment Report for SAB, Ford and Serdynska (2013), make more precise the ecological components that SAB might help to protect and conserve, in alignment with the Oceans Act's definition of MPAs:

\begin{itemize}
	\item Commercial and non-commercial fishery resources, including marine mammals, and their habitats (e.g., habitat for Atlantic cod, redfish, American plaice, sea urchins, white hake, witch flounder, sea anemones, sponges, and sea pens)
  \item Endangered or threatened marine species, and their habitats (e.g., habitat for depleted species such as Atlantic wolffish, Atlantic cod, and leatherback turtles )
  \item Unique habitats (it is the only major bank on the Inner Scotian Shelf)
  \item Marine areas of high biodiversity or biological productivity of invertebrates and fish
\end{itemize}

\begin{figure}[h]
  \label{fig:SABCloseup}
  \centering
  %\includegraphics[width=0.6\textwidth]{\analysis/maps/mpa_closeup.pdf}
  \includegraphics[width=0.6\textwidth]{\sab SAB_MPA.pdf}
  \caption{Bathymetic (100 m resolution) chart of the  St. Anns Bank area with the proposed St Anns Bank Marine Protected Area (thick maroon polygon) and limited fishing zones (maroon lined polygons). See Figure\ref{fig:SAB} for geographic location in a larger map.}
\end{figure}

\subsection{Objectives}

The primary objectives of this report are straightforward:

\begin{itemize}
	\item To develop an Assessment Framework that can:
  \begin{itemize}
    \item	Monitor the status of an MPA.
  	\item Assess the effectiveness of an MPA in meeting its conservation objectives
  \end{itemize}
  \item To identify data gaps and sources of uncertainty 
\end{itemize}

The method by which this can be accomplished, however, is anything but straightforward. This is primarily due to the fact that SAB is:

\begin{itemize}
	\item large ecosystem and as such complex, operating at various space, time and organizational scales
	\item Connected in various ways to the surrounding environment and so cannot be treated as an isolated system
	\item Measures of system components of interest, namely, productivity, biodiversity, habitat and species of interest, are imperfect at best, and usually non-existent or poor in information quality/quantity.
\end{itemize}

As such, we emphasize that this report is a simplistic first attempt at developing a general approach towards assessing MPAs, given the above challenges. It is best viewed as a work in progress that will require further precision and improvement.
To this end, we will in the following, describe the data used for the assessment (Section 2); outline the methods and assumptions associated with the modelling of this data (Section 3); summarize the primary results of this analysis (Section 4); provide some discussion of salient points (Section 5); and conclude with general recommendations (Section 6). The technical aspects of data QA/QC and associated assumptions are identified in Appendix A. 

\medskip

\textbf{NOTE: The primary purpose of this first document is upon Section 2 (data) and Section 3 (proposed methods). There are no Results nor Conclusions to report. }

\section{Data} 
In this section we focus upon a basic description of the data chosen for inclusion in this assessment. The purpose of the section is basic: to clearly identify the data, sampling design and the associated assumptions and methods required/used to filter and integrate them in an informative manner. For the sake of transparency, all data assimilation and QA/QC methods have been encoded in R \cite{rCran} and made publicly accessible via \url{https://github.com/jae0/ecomod/} . 


\subsection{Study area}
Any evaluation of MPA status and effectiveness in meeting conservation objectives, requires explicit reference to changes both within and without the area of interest. Even in the most basic BACI-type design, this requirement is explicit (Green 1979, Underwood 1992). Although this will not alleviate problems associated with pseudo-(spatial, temporal)-replication (Hurlbert 1984), it is still a minimum requirement. For this reason, and also to facilitate evaluations of other potential MPAs in the region, a much larger surrounding area was chosen for analytical purposes. This area is the continental shelf region off Nova Scotia (Figure~\ref{fig:SAB}), bounded by latitudes 37N to 48N and longitudes from 48W to 71W.  

\begin{figure}[h]
  \label{fig:SAB}
  \centering
  \includegraphics[width=0.6\textwidth]{\analysis/maps/mpa.pdf}
  \caption{Map of the data extraction area 37N to 48N and from 48W to 71W and the relative location of the St Anns Bank MPA.}
\end{figure}


\subsection{Data selection criteria}
Exhaustive surveys of available data have been compiled by Ford and Serdynska (2013). Their conclusions were that most biological data and environmental conditions are poorly sampled in the SAB area. 
The decision criteria for inclusion of data in this study was as follows:

\begin{itemize}
	\item Part of an \textbf{on-going sampling }program. This is because the design principle behind this project is that the underlying  assessment will become a routine monitoring approach into the future.
  \item	Sufficient and regular \textbf{spatial} coverage ( $>$ XX sampling locations ) inside MPA and throughout the study area
  \item Sufficient and regular \textbf{temporal} coverage ( \~{} annual, $>$ XX years ) inside MPA and throughout the study area
  \item \textbf{Informative} ( i.e., high data quality ) in some manner to productivity, biodiversity, habitat and species of interest
\end{itemize}

The same decision criteria were applied to human usage data. The result was to include the following data streams for MPA characterisation:

\begin{itemize}
	\item Chlorophyll-a and Nutrients: BioChem bottle data (AZMP, DFO 2015, Devine \textit{et al}. 2014) 
  \item Zooplankton: BioChem database (AZMP, DFO 2015, Devine \textit{et al}. 2014) 
  \item Remote Sensing Data: ocean colour and SST (Remote Sensing Group)
  \item Groundfish: DFO's Groundfish Research Vessel Surveys focus upon demersal fish species, since $\approx$ 2000, upon invertebrates as well
  \item Snow crab surveys ( \ldots{} ) focus upon benthic invertebrates
  \item Clam survey data in Banquereau and Western Banks (though it does not pass the temporal coverage conditions, it offers very high resolution multispecies data on the banks)
  \item Temperature records: from various sources, especially, Groundfish, Snow crab and AZMP surveys
  \item Salinity (Groundfish surveys/AZMP, BioChem)
  \item Oxygen and pH (once the data have been reloaded)
  \item Bathymetry (CHS, Groundfish surve, Snow crab survey )
\end{itemize}

To characterise human usage patterns, the following have been chosen for inclusion:

\begin{itemize}
	\item Logbook records of catch and effort (MarFIS)
  \item AIS tracks
\end{itemize}

\subsection{Discrete Bottle Data: Chlorophyll-a and Nutrients}

\begin{itemize}
  \item Relevance:  productivity, biodiversity, habitat and species of interest (in relative order)
  \item Sampling:  AZMP surveys, Groundfish surveys, pelagic net tows and water profiles
  \item Spatial coverage: XXX stations, 143,499 records, 829 missions
  \item Temporal coverage:,  1955 to present, annual surveys
  \item Source code: \url{https://github.com/jae0/ecomod/biochem/src/biochem.r}
\end{itemize}

Discrete bottle data consisting of chlorophyll-a and nutrient records (nitrate, phosphate and silicate) were obtained by laboratory analysis of water samples collected at discrete depths. For this study all available nutrient and chlorophyll-a discrete bottle data was extracted from DFO's BioChem database for the study area. This dataset contains more than 500 000 records with the earliest records starting in 1955. After QA/QC, the discrete bottle data retained for analysis was comprised of 143 499 profiles, collected on 829 missions (Figure~\ref{fig:BottleMap}).



The number of profiles available in each year (Figure~\ref{fig:ChloroMap}) shows that there were few profiles taken until the mid 60's, and a relatively steady number of yearly profiles after the initiation of the Atlantic Zone Monitoring Program (AZMP) in 1999.The peak sampling during the period 1976-1982 corresponds to DFO's Scotian Shelf Ichthyoplankton Program (SSIP) and foreign research vessels sampling programs which were obtained from NOAA's National Oceanographic Data Center (Pierre Clement, personal communication).  Monthly distribution of the profiles (Figure~\ref{fig:ChloroFreq})  demonstrates that most of the data was collected in July (mostly during DFO's groundfish surveys), followed by the months of September and April. Note that spatial distribution of the sampling varies among months with most data collected on the Scotian Shelf in July and the fewest data in January (Figures~\ref{fig:BottleMap}, \ref{fig:ChloroProfiles}.)


\begin{figure}
  \label{fig:ChloroMap}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 2.png}
  \caption{Number of chlorophyll and nutrient profiles extracted from the BioChem database for each year since 1955.}
\end{figure}

\begin{figure}
  \label{fig:ChloroFreq}
  \centering
  \includegraphics[width=0.8\textwidth]{\sab 3.png}
  \caption{Number of chlorophyll and nutrient profiles extracted from the BioChem database for the time period 1955-2014, grouped monthly.}
\end{figure}

\begin{figure}[h]
  \label{fig:BottleMap}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 4.png}
  \caption{Monthly spatial distribution of discrete bottle data for the time period 1955-2014.}
\end{figure}

\begin{figure}
  \label{fig:ChloroProfiles}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 5.png}
  \caption{Depth profiles of chlorophyll-a and nutrients; all data for the time period 1955-2014.}
\end{figure}


2.3.2  Zooplankton Data

\begin{itemize}  
  \item Relevance:  productivity, biodiversity, species of interest, habitat (in relative order)
  \item Sampling:  AZMP surveys, Groundfish surveys, pelagic net tows, 400 taxonomic species
  \item Spatial coverage: 2367 net deployments, 126 missions 
  \item Temporal coverage:,  1999 to 2014, annual surveys
  \item Source code: https://github.com/jae0/ecomod/biochem/src/biochem.r
\end{itemize}

Number of net deployments for each month is shown in Figure~\ref{fig:AZMPdeploymentsMonthly} and the corresponding spatial distribution of the net deployments are shown in Figure~\ref{fig:AZMPdeploymentsMonthlyMap}. Note that most of the net data was collected in July during summer groundfish survey missions and in April and October on AZMP spring and fall missions while winter months contain mostly fixed station data (Halifax 2 and Prince 5).

Abundance patterns are found in Figure~\ref{fig:AZMPBiomassMonthly}.

\begin{figure}[h]
  \label{fig:AZMPdeploymentsMonthly}
  \centering
  \includegraphics[width=0.8\textwidth]{\sab 6.png}
  \caption{Total number of net deployments for each month during the time period 1999-2014.}
\end{figure}

\begin{figure}[h]
  \label{fig:AZMPdeploymentsMonthlyMap}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 7.png}
  \caption{Spatial distribution of net deployments included in the zooplankton dataset. }
\end{figure}

\begin{figure}[h]
  \label{fig:AZMPBiomassMonthly}
  \centering
  \begin{tabular}{cc}
      \includegraphics[width=0.45\textwidth]{\sab 8.png}
      \includegraphics[width=0.45\textwidth]{\sab 9.png}
  \end{tabular}
  \begin{tabular}{cc}
      \includegraphics[width=0.45\textwidth]{\sab 10.png}
      \includegraphics[width=0.45\textwidth]{\sab 11.png}
  \end{tabular}
  \caption{Monthly averages of all data from 1999 to 2014: total abundance (top left), total biomass computed from wet weight (top right), ratio of total biomass computed from wet weight  to total abundance (bottom center) as a potential measure of the average weight of the individual organism, \textit{Calanus finmarchicus}, \textit{Calanus hyperboreus}, and \textit{Calanus glacialis} }
\end{figure}

\clearpage

\subsection{Remote Sensing Data} 

\paragraph{Ocean Colour}

\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest (in relative order) 
  \item Sampling:  MODIS
  \item Spatial coverage: 39 N to 62.5 N and 42 W to 71 W, resolution of 1.5 km
  \item Temporal coverage: August 2002 to March 2015, 610 quarter-monthly (8-day) composite  images
  \item Source code: \url{https://github.com/jae0/ecomod/remote.sensing/src/remoting.sensing.r}
\end{itemize}

Ocean colour refers to the spectral distribution of light emerging from the ocean which carries information about water constituents, particularly about biologically useful chlorophyll concentration in the surface layer. When measured from satellites it provides unique synoptic view of chlorophyll spatial distribution over large areas of the ocean on a daily time scale.

The nominal uncertainty of chlorophyll products derived from ocean colour satellites is 35\%, with better agreement with in-situ chlorophyll for the open ocean (Moore \textit{et al}. 2008) while overestimation is often observed in the coastal ocean (Darecki and Stramski 2004). This is due to the inability of the algorithms to distinguish chlorophyll from suspended particulate matter and colored dissolved organic matter often present in the coastal waters, as for example in the Bay of Fundy and Northumberland Strait.

Ocean colour satellite data for this study was provided by the Remote Sensing Unit (RSU) from the Bedford Institute of Oceanography (DFO) as 8-day composite chlorophyll images which are routinely produced by the unit for the Atlantic Zone Monitoring Program (AZMP). The dataset was created using the Moderate Resolution Imaging Spectroradiometer (MODIS-Aqua) data, where the chlorophyll-a values are based on the 2012 reprocessing carried by the National Aeronautics and Space Administration (NASA) using OC3 chlorophyll algorithm. Composite images were created from daily Level-2 MODIS-Aqua files downloaded from NASA by averaging valid chlorophyll-a values for each pixel using all available daily images within that time period (RSU Technical document 5). The dataset comprises years 2002 to 2015 and area 39 N to 62.5 N and 42 W to 71 W, with spatial resolution of about 1.5 km per pixel. 

Even though there is ocean color data available before MODIS launch it was decided to limit our dataset to a single sensor to avoid potential biases between the sensors. Due to the frequent cloud coverage of the NW Atlantic it was decided to use 8-day composite images as daily images would not provide sufficient number of valid pixels. 

The values of chlorophyll-a pixels within St. Anns Bank polygon were extracted from each 8-day composite image and average chlorophyll-a concentration was computed for the polygon. The time series for the polygon and the associated climatology that was computed from time series data show characteristic spring blooms in March and April, with varying intensity and timing throughout the years (Figures~\ref{fig:modisChlaTS},~\ref{fig:ChlaSeasonal}). An example of MODIS semi-monthly chlorophyll-a products showing spring bloom progression in the St. Anns Bank area in 2012 is shown on Figure~\ref{fig:MapChlaBloomSpring}.

\begin{figure}[ht]
  \label{fig:MapChlaBloomSpring}
  \centering
  \begin{tabular}{cc}
    \includegraphics[width=0.45\textwidth]{\sab 12.jpg}
    \includegraphics[width=0.45\textwidth]{\sab 13.jpg}
  \end{tabular}
  \begin{tabular}{cc}
    \includegraphics[width=0.45\textwidth]{\sab 14.jpg}
    \includegraphics[width=0.45\textwidth]{\sab 15.jpg}
  \end{tabular}
  \caption{MODIS semi-monthly Chl-a concentration showing spring bloom progression in the NW Atlantic in 2012. Note the intense bloom at St. Anns Bank during the last two weeks in March.}
\end{figure}


\begin{figure}[h]
  \label{fig:modisChlaTS}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 16.png}
  \caption{Chlorophyll-a concentration extracted from MODIS 8-day composite images for St.Anns Bank polygon for the time period 2002-2015.}
\end{figure}


\begin{figure}[h]
  \label{fig:ChlaSeasonal}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 17.png}
  \caption {Average Chorophyll-a concentration computed from 8-day composite images for St.Anns bank polygon for the time period 2002-2015. }
\end{figure}



\paragraph{Primary Production}

\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest (in relative order)
  \item Sampling:  MODIS, ...
  \item Spatial coverage: 39 N to 62.5 N and 42 W to 71 W, resolution of 1.5 km
  \item Temporal coverage: July 2002 to December 2014, 150 monthly  images
  \item Source code \url{https://github.com/jae0/ecomod/remote.sensing/src/remoting.sensing.r}
\end{itemize}



Marine primary production plays an important role in biogeochemical cycles, in food web dynamics, and in marine fisheries. It may be defined as the amount of organic material (or organic carbon) produced per unit area (or volume) per unit time by photosynthetic plants, predominately by phytoplankton.

Primary production of the ocean on synoptic scale is estimated by models that use satellite data (SST, ocean colour chlorophyll, and available light estimates at the surface), shipborne in-situ information on the vertical distribution of phytoplankton in the water column, and the phytoplankton’s photosynthetic response to light.

Monthly primary production data were provided by the Remote Sensing Unit (RSU) at BIO that routinely generates production maps for the NW Atlantic as part of the Atlantic Zone Monitoring Program. The general approach for the production computation is described in Platt et al. (2008) and employs chlorophyll and light estimates from MODIS, SST produced by the unit, and DFO’s archive of shipborne measured parameters. This particular production algorithm has been validated with in-situ measured production (Platt et al., 1988) and also has performed very well in global comparisons (Carr et al., 2006).

Primary production for each pixel within St. Anns Bank polygon was extracted from the monthly images and average production was computed for the polygon. The time series for the polygon and the associated monthly climatology are showing peaks in primary production in spring and summer with varying intensity and timing throughout the years (Figure~\ref{fig:ppTSmonthly} and Figure~\ref{fig:ppTSannual}). 


\begin{figure}[h]
  \label{fig:ppTSmonthly}
  \centering
  \includegraphics[height=8.326cm,width=16.51cm]{\sab 18.png}
  \caption{Annual monthly Primary Production (PP) computed from PP composite images for St.Anns bank polygon for the time period 2002-2014.}
\end{figure}


\begin{figure}[h]
  \label{fig:ppTSannual}
  \centering
  \includegraphics[height=8.702cm,width=15.571cm]{\sab 19.png}
  \caption{Average Primary Production (PP) computed from monthly composite images for St.Anns bank polygon for the time period 2002-2014.}
\end{figure}

\clearpage

\paragraph{Sea Surface Temperature}


\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest (in relative order)
  \item Sampling:  AVHRR
  \item Spatial coverage: 39 N to 62.5 N and 42 W to 71 W, resolution of 1.5 km
  \item Temporal coverage: December 1997 to March 2015, 845 8-day composite images
  \item Source code: \url{https://github.com/jae0/ecomod/remote.sensing/src/remoting.sensing.r}
\end{itemize}

Sea Surface Temperature (SST) from space is measured using infrared channels of the Advanced Very High Resolution Radiometer (AVHRR) on board the polar-orbiting satellites.

The SST data for this study was provided by the Remote Sensing Unit from the Bedford Institute of Oceanography (BIO) that has been downlinking AVHRR data from the National Oceanographic Atmospheric Administration (NOAA) satellites since 1997 on an L-band satellite receiver that resides on the roof of the BIO. The received data is routinely processed by the RSU and is supplemented by the data stream from the AVHRR onboard the European satellites. Composite SST images of different time periods are operationally produced as part of the Atlantic Zone Monitoring Program (RSU Technical Document 1). Here we used 8-day composite images with the same spatial coverage and spatial resolution as the ocean colour chlorophyll data.

The SST pixels within St. Anns Bank polygon were extracted from each 8-day composite image and average SST was computed for the polygon. The time series for the polygon and the associated climatology that was computed from time series data are shown on Figures~\ref{fig:SSTfromAVHRRmap} and \ref{fig:SSTfromAVHRRts}. An example of semi-monthly SST products corresponding to the spring bloom progression in the St. Anns Bank area in 2012 is shown in Figure~\ref{fig:SstSeasonal}.

\begin{figure}[h]
  \label{fig:SSTfromAVHRRmap}
  \centering
  \begin{tabular}{cc}
    \includegraphics[width=0.5\textwidth]{\sab 20.jpg}
    \includegraphics[width=0.5\textwidth]{\sab 21.jpg} 
  \end{tabular}
  \begin{tabular}{cc}
    \includegraphics[width=0.5\textwidth]{\sab 22.jpg}
    \includegraphics[width=0.5\textwidth]{\sab 23.jpg}
  \end{tabular}
  \caption{Bi-weekly composites from AVHRR showing SST in the North West Atlantic in the spring of 2012, corresponding to the intense spring bloom at St.Anns bank shown in Figure~\ref{fig:ChlaSeasonal}.}
\end{figure}

\begin{figure}[h]
  \label{fig:SSTfromAVHRRts}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 24.png}
  \caption{Sea Surface Temperature (SST) extracted from 8-day AVHRR composite images for St.Anns bank polygon for the time period 1997-2015.}
\end{figure}

\begin{figure}[h]
  \label{fig:SstSeasonal}
  \centering
  \includegraphics[width=0.8\textwidth]{\sab 25.png}
  \caption{Average Sea Surface Temperature (SST) computed from 8-day AVHRR composite images for St.Anns bank polygon for the time period 1997-2015.}
\end{figure}

\clearpage


\subsection{Bottom temperatures}

\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest
  \item Sampling:  Groundfish survey, snow crab survey, AZMP profiles 
  \item Spatial coverage: full extent, varied sampling, XXX stations
  \item Temporal coverage: 1950 - present
  \item Source code: https://github.com/jae0/ecomod/temperature/src/temperature.r
\end{itemize}

Numerous data sources have been compiled by Ocean Sciences Division. The data were QA/QC controlled and then modelled in a two-part process, temporal (first order harmonic analysis) and then simple spatial interpolations via localized Generalized Additive Models with a spatial extent that varied as a function of data density. 

\clearpage

\subsection{Demersal fish and macro-invertebrates}
\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest
  \item Sampling:  Groundfish survey, Snow crab survey 
  \item Spatial coverage
    \begin{itemize}
      \item Groundfish: full extent, random stratified, \~{} 200?? stations
      \item Snow crab: Colder water environment, geostatistical grids of \~10 minutes, \~400 stations
    \end{itemize}
  \item Temporal coverage
    \begin{itemize}
      \item Groundfish: 2000 - present (started in 1970, but consistent sampling since 2000)
      \item Snow crab: 2005 - present (started in 1996, but consistent sampling since 2005)
    \end{itemize}

  \item Source code
      \begin{itemize}
        \item https://github.com/jae0/ecomod/groundfish/src/groundfish.r
        \item https://github.com/jae0/ecomod/snowcrab/src/1.snowcrab.r
      \end{itemize}
  
\end{itemize}


\subsubsection{Groundfish survey}

The Groundfish survey utilizes a Western II-A net ...config ..
A standard tow is a 30 minute tow at xx knots.
It operates night and day.


\subsubsection{Net mensuration} 

The groundfish survey is still a major source of information for many studies and stock assessments. However, historical estimation procedures were incorrect due to timing/net mensuration methods being inadequate. Sensors measuring trawl net configuration and state are used by most modern surveys. For example, in the snow crab survey, they have been used to determine swept area manually (since 1996) and/or with an automated procedure (since 2004).

In the groundfish survey, net configuration data have been collected sporadically since 2004. However, swept area estimates have never been computed for the groundfish surveys. Generally, data seem to be used with the assumption that each tow is equivalent in swept area or adjusted by the linear distance from start and end positions (assuming a fixed net width). The latter approach is problematic (Figures~\ref{fig:groundfishWingDoorAnnual}, \ref{fig:groundfishWingDoorComparison}). Recorded positional information was also found to be recorded with error (Figure~\ref{fig:groundfishTowDistance} resulting in appropriate estimates of swept area (Figure~\ref{fig:groundfishSweptArea}. Further, the actual time and distance of bottom contact was variable ranging from approximately 10 to 40 minutes and with distances ranging from xx to xx km. Net width always assumed constant at xx m. This assumption is also problematic in that the configuration of a trawl net can alter shape depending upon currents, surface sea state, net fullness/filtration efficiency due to contact with rocks/boulders/mud, etc. (Munden and Choi, in prep). 

To address these important and incorrect assumptions, algorithms were developed to automatically determine lift off and touch down times, locations and net width. Based upon random visual inspection, the skill was determined to be high with approximately 95\% of the cases having estimates within 15 seconds of a visual determination touch down/lift off points. Where bias was observed this was mostly to underestimate total contact time due to an over-smoothing of the lift off or touch down profiles. Unfortunately, net mensuration data is still not consistently recorded nor used in the groundfish surveys (Table XX -- number of sets with records of net config). As a result, it is not possible to satisfactorily estimate swept area for all historical sets. As a result, statistical methods have been developed to estimate these as much as possible and are used for the first time in this report. The estimation procedure was statistical using GAM-estimated relationships with location, depth and salinity ($R^2$ ranged from 60 to 40\%, depending upon availability of covariates). 

\begin{figure}[h]
  \label{fig:groundfishTowDistance}
  \centering
  \includegraphics[width=1\textwidth]{\ecomod groundfish/analysis/toweddistance.pdf}
  \caption{Towed distance comparisons in the groundfish survey.}
\end{figure}


\begin{figure}[h]
  \label{fig:groundfishWingDoorAnnual}
  \centering
  \includegraphics[width=1\textwidth]{\ecomod groundfish/analysis/{wing.v.door.byyear}.pdf}
  \caption{Net spread variations by year. Note in 2011, the doorspread sensors seem to have failed completely. Note also that wingspread has been significantly larger from 2013 to 2015.}
\end{figure}


\begin{figure}[h]
  \label{fig:groundfishWingDoorComparison}
  \centering
  \includegraphics[width=1\textwidth]{\ecomod groundfish/analysis/{wing.v.door}.pdf}
  \caption{Net spread variations: doorspread vs wingspread. Note also that wingspread has been significantly larger from 2013 to 2015 but not doorspread.}
\end{figure}


\begin{figure}[h]
  \label{fig:groundfishSweptArea}
  \centering
  \includegraphics[width=1\textwidth]{\ecomod groundfish/analysis/{sa.comparison}.pdf}
  \caption{Surface area estimates based on GSINF logged start-end positions vs computed surface area estimated from tow track and net configuration.}
\end{figure}



Trawl data summary:

-- add time and location information

-- suggest utility for mutlispecies to start in 2010 (groundfish) and 2005 (snow crab)

-- describe use of quantiles / Zscore

\clearpage


\subsubsection{Snow crab survey}
\label{(se:snowcrab)}
A \textit{Bigouden Nephrops} trawl, a net originally designed to dig into soft sediments for the capture of lobsters in Europe was used to sample the substrate (headline of 20~m, 27.3~m foot rope mounted with a 3.2~m long 8~mm chain, with a mesh size of 80~mm in the wings and 60~mm in the belly and 40~mm in the cod-end). Tows were conducted for $\sim$~5~minutes in duration with actual duration of bottom contact being monitored by Netmind sensors. The width of the mouth of the net was also monitored with Netmind sensors. The ship speed was maintained at $\sim 2$~knots. The warp length was $\sim 3 \times$ the depth. Positional information as well as water temperature measurements were collected using a global positioning system and Minilog data recorders, respectively. The surface area swept by the net was calculated from swept distance and net width information.

\subsubsection{MPA stations}

The 2015 snow crab trawl survey increased sampling in the St. Anns Banks area to provide additional information about the marine macro-fauna. Fourteen stations in (or adjacent to) the proposed MPA location were included in this additional sampling. These fourteen stations were locations traditionally sampled on the snow crab survey and represent varied depths and bottom- types, micro-habitats. The species composition of the catch at these stations varied greatly as expected with differences in depth and bottom type. The sampling at these stations included:

\begin{itemize}
  \item all catch identified to species level
  \item all species counted and weighed to tenth of a kilogram
  \item all finfish and crab species measured and weighed individually
  \item stomach samples taken from fin-fish for diet studies
\end{itemize}

An overview (in Google Earth format) of the catch and sampling at these stations can be found at: http://www.enssnowcrab.com/mpa/mpatows.kmz

\subsection{Other ecosystem data not being considered}
\begin{itemize}
   \item Halibut survey - hook methodology and associated bias, hook saturation, targetting only large species, by catch not complete 
   \item 4Vn Sentinel survey .. very location specific
   \item including benthic sampling in the AOI, 
   \item bird observations carried out by Canadian Wildlife Service, 
\end{itemize}	
  
\clearpage

\subsection{Fishery activity}
Commercial-fishing activities can modify the habitat and ecosystem and contribute to changes in the structure and functioning of exploited marine communities.  Fishing impacts can be direct, such as the reduction of targeted and non-targeted species, as well as their truncations in age and size compositions.  Other direct effects due to fishing activities include habitat alterations and substrate modifications through interactions with fishing gear. Fishing can also cause indirect impacts with changes foodweb structure within an ecosystem.  

Direct impacts of commercial fishing can be measured using data form the Marine Fish (MARFIS) database that provides information on commercial fishing activities.  For most fisheries the fishing position, gear type, catch per unit effort, estimated weight of catch by species information is available from the database. The MARFIS database detail information for all fishing trips where a landing is reported within the DFO Maritimes region and includes data from 2002 through 2015.  

\begin{itemize}
  \item Relevance:  productivity, habitat, biodiversity and species of interest
  \item Sampling:  Marfis
  \item Spatial coverage: full extent, XXX ?
  \item Temporal coverage: XXXX - present
  \item Source code: https://github.com/jae0/ecomod/XXX/src/XXX.r
\end{itemize}	




St. Anns Bank Area of Interest has four zones within it with various levels of fishing restrictions (Figure \ref{fig:SABCloseup}). Commercial fishing is restricted from Zone 1, the largest area, with the exception of the seal harvest.  The MPA requires monitoring to ensure that fishers are complying with regulations.  Monitoring could be done through a variety of techniques and data sources, including data reported in the MARFIS database, Vessel Monitoring System (VMS) data that allow the direct monitoring of fishing activities. Automatic Identification System (AIS) data may also be used to monitoring fishing activities if the vessel is large enough to require AIS ($\geq$ 300 gross registered tonnes) or if a fishing vessel has voluntarily installed an AIS system onboard.  


\subsubsection{MarFIS data extraction and manipulation}

In consultation with Commercial Data Division staff (Robert Grandy, Greg Croft and Krista Wry), prorated landings for all species from 2002 onwards were extracted from MARFIS. The proration process distributes the actual reported weights across the reported efforts as identified within the fishers' logs.

Most logs include coordinates describing the location of the catch, but it is not uncommon to also find those that don't.  Coordinates were used preferentially in the following order:
\begin{itemize}
  \item LOG\_EFRT\_STD\_INFO\_ID.ENT\_
  \item LOG\_EFRT\_STD\_INFO\_ID.DET\_
  \item Centroid of the determined (NAFO) Fishing Area
\end{itemize}

Those catches that did not have associated coordinates were identified so that significant catches would not be accidentally overlooked. Catches without associated coordinates or those falling on land will be spread over the whole spatial area of the species, with weights distributed in proportion to known catch locations for a given year.

In addition to the landings, we have also included several other forms of catch so as to better reflect the removal of biomass. These catches include those used as bait or discarded (including live, dead and unspecified). The position logic was the same as that used above. Additionally, these catches are self-reported in a variety of units, so they are converted to kg as necessary to match the logged landings The catch consisting of "live discards" will not be included in the final report.

The extraction logic is captured in the script found at: \\

 url{https://github.com/jae0/ecomod/mpa/src/\_Rfunctions/get.marfis.data.R}

\subsubsection{Data Layer Generation}

Data was aggregated at a 6 minute grid, and individual polygon files were created for each unique combination of YEAR and SPECIES. (Note: The aggregation will ultimately be at 3 minute grids, which was selected since it is an even division of a degree, with no potential for rounding errors: 

\begin{eqnarray*}
3 \: \text{minutes} &:& 1/60*3 = 0.05 \\
2 \: \text{minutes} &:& 1/60*2 = 0.0\dot{3}
\end{eqnarray*}


The data aggregation logic is captured in 2 scripts: \\

url{https://github.com/jae0/ecomod/mpa/src/\_Rfunctions/process.marfis.mpa.R} \\
url{https://github.com/jae0/ecomod/marfissci/src/\_Rfunctions/aggregate.marfis.R}


\clearpage

\subsection{Vessel activity}
Commercial shipping can have various direct and indirect effects on an ecosystem. Direct effects including the contamination of the ecosystem from the discharge of contaminates, radiated underwater noise, the introduction of aquatic invasive species, and vessel-strike risk to marine mammals and sea turtles.  Spatia-temporal data on vessel traffic is needed to determine the probability and/or magnitude of these effects on ecosystems.  Such information is available through AIS data.  

The International Maritime Organization (IMO) requires AIS transponders on all international vessels $\geq$ 300 gross tonnage and all passenger vessels.  Many studies have used AIS data to examine risk of lethal vessel collisions to large whales (e.g. Vanderlaan and Taggart 2009, Wiley \textit{et al}. 2011, Redfern \textit{et al}. 2013,  Guzman \textit{et al}. 2013) or to assess and monitor ship noise and assess the impact on marine mammals (Hatch \textit{et al}. 2008 McKenna \textit{et al}. 2012,  Hatch \textit{et al}. 2012, Merchant \textit{et al}. 2014).    Similar exercises can be done in the proposed St. Anns Bank MPA with AIS data. 

Fisheries and Oceans Canada has access to two different sources of AIS data.  The first is from the Canadian Coast Guard (CCG) terrestrial system that was developed to track and monitoring coastal shipping and provides a real-time, continuous stream of AIS vessel positions.  Archived historical data from this system is available for January 2012 through December 2015 and data from 2016 is currently streaming and archiving to a server within DFO. Decoding routines have been developed using native R methods for these data.   Both sources of data provide dynamic and static data, where the dynamic data includes information on vessel identity, speed, and location, and static data includes information vessel identity, name, size, and type.   

The CCG terrestrial system as 21 AIS coastal receiving stations in in the Maritime region and 19 AIS coastal receiving stations in Newfoundland and Labrador.  These receiving stations have limited detectability (Figure~\ref{fig:TAIS}) as AIS transmission detectability are primarily a function of the receiving tower\textquoteright height above sea level and the height of the AIS antenna on the transmitting vessel.  AIS data is transmitted via Very High Frequency (VHF) marine radio on two channels (161.9765 and 162.025 MHz).  Based on the height of the associated towers and a vessel with an AIS antenna 100 m high, line of sight calculations for VHF provide a reception range of  57 to 113 m (Figure~\ref{fig:LOF}).  However there are several other factors that will contribute to transmission range including weather conditions and the height of the AIS transmitter aboard each vessel.  Simard \textit{et al}. (2014) estimated that coastal antennas within this network generally provide a reception range of ~100 km (Figure~\ref{fig:DetAIS}).  In either case the terrestrial network is insufficient to monitor vessels across the entire AOI and just north of the AOI. This data can be combined with satellite AIS data, available from exactEarth.


\begin{figure}[h]
	\label{fig:TAIS}
  \centering
	\includegraphics[width=1.0\textwidth]{\sab Line_of_Sight.pdf}
	\caption{AIS data collected from the Canadian Coast Guard terrestrial network of AIS receiving stations.  Each colour represents data from a unique vessel.}
\end{figure}


\begin{figure}[h]
	\label{fig:LOF}
	\centering
	\includegraphics[width=1.0\textwidth]{\sab Line_of_Sight.pdf}
	\caption{Bathymetic (100 m resolution) chart of the  St. Anns Bank area with line of sight detection (red circles) for the terrestrial AIS receiving stations (red dots) around St. Anns Bank Area of Interest.}
\end{figure}

\begin{figure}[h]
	\label{fig:DetAIS}
	\centering
	\includegraphics[width=1.0\textwidth]{\sab Simard_Sight.pdf}
	\caption{Bathymetic (100 m resolution) chart of the  St. Anns Bank area with Simard \textit{et al}. (2014) estimated vessel detection distances (blue circles) for the terrestrial AIS receiving stations (blue dots) around St. Anns Bank Area of Interest}
\end{figure}


The satellite AIS data are available globally for the years 2013, 2014, and 2015 with on-going data collection for 2016. Although satellite AIS data coverage if global, data are limited temporally as large sections of vessel transits are unavailable due (see Figure~\ref{fig:astar})to a limited number of satellites (n = 8) and their orbital paths.  Spatial interpolation must be completed to \textquotedblleft fill in\textquotedblright  missing data.  Spatial interpolation is achieved using an astar function (Hart \textit{et al}. 1968)  that estimates the minimum cost to get from one point to another based on a cost map.  The coast map was estimated from seasonally aggregated annual density distributions of satellite AIS data for the years 2013 through 2015 (Figure~\ref{fig:costmap}).  Grid resolution for this analysis was set to 0.01 degrees and within each grid square the number of unique vessels, identified by the vessel's Maritime Mobile Service Identity (MMSI) number was counted daily and summed through time.  Cost maps were estimated quarterly.  Two different cost maps were developed for the St. Anns Bank area to interpolate vessel transits north of Cape Breton into and out of the Gulf of St. Lawrence.  Interpolation was heavily influenced by the ferries transiting between Cape Breton and Newfoundland, and therefore a cost map was developed without the data derived from these ferries.  A bathymetric restriction can also be build into the cost maps.

\begin{figure}[h]
	\label{fig:astar}
	\centering
	\begin{tabular}{cc}
	\includegraphics[width=0.5\textwidth]{\sab Interpolation_Example.pdf} &
  \includegraphics[width=0.5\textwidth]{\sab Interpolation_Example3.pdf}
	\end{tabular}
	\caption{Detected vessel positions (large filled circles) and interpolated vessel positions (lines) for three unique vessels transiting through the St. Anns Bank Area, where each colour represents a unique vessel. The left panel represents interpolated vessel position where a bathymetric restriction of 20 m was imposed on the astar function that interpolates positions based on cost maps.}
\end{figure}


\begin{figure}[h]
	\label{fig:countmaps}
	\centering
	\begin{tabular}{cc}
		\includegraphics[width=0.5\textwidth]{\sab Counts_2013-2015_Q1.pdf} &
		\includegraphics[width=0.5\textwidth]{\sab Counts_2013-2015_Q1_NNF.pdf}
	\end{tabular}
	\caption{Vessel density maps for the first quarter of a year based on satellite AIS data from 2013-2015 for all vessels (left panel) and all vessels except of the Newfoundland ferries (right panel).}
\end{figure}




\begin{figure}[h]
	\label{fig:costmap}
	\centering
	\begin{tabular}{cc}
		\includegraphics[width=0.5\textwidth]{\sab Cost_2013-2015_Q1.pdf} &
		\includegraphics[width=0.5\textwidth]{\sab Cost_2013-2015_Q1_NNF.pdf}
	\end{tabular}
	\caption{Cost maps developed for the astar function to interpolate undetected vessel positions as vessels transit in and out of the Gulf of St. Lawrence.}
\end{figure}




\begin{itemize}
  \item Relevance:  habitat, biodiversity and species of interest, (productivity?)
  \item Sampling:  AIS
  \item Spatial coverage: full extent, data density?
  \item Temporal coverage: XXXX - present
  \item Source code: https://github.com/jae0/ecomod/AIS/src/ais.r
\end{itemize}	


\subsection{Other human usage data not being considered}
\begin{itemize}
  \item human activities::
  \item other Fishing: 
  \item Snow crab pot\\Lobster pot\\Otter trawl (e.g., for redfish and potentially cod)\\Redfish midwater trawl\\Groundfish seine\\Halibut longline\\Herring or mackerel gillnet 
  \item cnsopb /oil and gas ?Seismic +Drilling (exploratory or production) 
  \item pollution monitoring ?
  \item Ballast water exchange 
  \item Recreation and tourism
  \item Research surveys
  \item Debris (garbage, persistent litter) 
  \item other ?
\end{itemize}	
  
\subsection{Data gaps}
\begin{itemize}
  \item pelagic system:: fish, zoopl, nutrients 
  \item substrate characterisation ?  / multibeam data for whole shelf ?
  \item demersal fish -- high density/coherent survey
  \item marine mammals, reptiles, birds
  \item benthic / pelagic coupling and nutrient cycling
  \item genetic diversity 
  \item vulnerable life-history traits
  
\end{itemize}	

\subsubsection{Stomach Database}
Bottom trawl surveys have been conducted by DFO on the Scotian Shelf annual since 1970 using a stratified random design.  Sampling protocols changed in the late 1990s with the focus changing from commercially important finfish species to more comprehensive ecosystem monitoring that included the sampling of macroinvertebrates (Tremblay \textit{et al}. 2007).  Stomach contents samples were collected from finfish using a length-stratified sampling protocol.  Prey were quantified by weight and number, and often identified to the genus or family level, or to the species level when possible.  To determine if diet of finfish was different pre- and post- implementation of the MPA on St. Anns Bank or if there were differences within the MPA compared to other areas on the Scotia shelf, we explored the stomach database to determine if these dietary differences could be detected. Within the database, 54\% of the prey number observations were missing.  Due to the large interannual variation in prey weights estimating the prey number consistently from non-missing data where both the prey number and weight was available was determined to be impracticable and unreliable.  Due to the sampling stratification both by depth and length classes it was further determined that there would be insufficient samples available to reach the asymptote of prey species accumulation curves (Cook and Bundy 2010) and total diet composition could not be detected.  Prey species identified in the stomach samples could be used for quantifying biodiversity and species richness (Cook and Bundy 2012) for the proposed MPA and comparing it to similar ecosystem or pre- and post-implementation.


\clearpage
\section{Methods}

All methods have been and will continue to be implemented in R, an open-sourced programming environment. The methods are shared via http://github.com using the git revisioning system. These architectural choices were adopted to enhance the transparency and ease of sharing and collaborating with all interested parties. It is structured such that any additional data series can be easily added to the system to permit adaptive change. In this way, the approaches developed represents a true \textbf{structural framework} in which to further develop methods and approaches. 

The main methods used/developed in this report will be described in this section.


\subsection{Biodiversity and taxonomic diversity}


Biodiversity is seemingly a simple concept that in fact fundamentally complex. It ranges in focus from genetic and phenotypic varition within a local population, breeding populations to biome or even larger scaled genetics and phylogentic variability, both in terms of their number and relative dominance. As the spatial and temporal scales of the St. Anns Bank AOI is more or less constrained, we simplistically focus upon the number of unique species found in a given location (commonly called species richness). However, as this number is a function of sampling intensity, a statistical correction ("rarefaction") for spatial and temporal sampling intensity must be applied to be meaningfully comparable. Specifically, the model was:

\[
R = B0 + SA + TS + LN(e)
\]

The intent is to model the spatial/temporal patterns (Section~\ref{sec:interpolation}) and then integrate them in a risk-based approach (Section~\ref{sec:riskapproach}) to permit formal statements of risk and probability of exceeding thresholds. 

\clearpage
\subsection{Productivity}

Total system standing biomass is generally used as a proxy for productivity. They are not the same. But ...

- prod: total bottom, invert, pelagic zoopankton, chl, benthic fish?


This is an approximation based upon sampling from groundfish and snow crab surveys. 

This can then be coupled with total system landings to estimate the biomass and secondary production associated with the biota and fishery exploitation/footprint. 

The intent is to model the spatial/temporal patterns (Section~\ref{sec:interpolation}) and then integrate them in a risk-based approach (Section~\ref{sec:riskapproach}) to permit formal statements of risk and probability of exceeding thresholds. 


\subsection{Habitat}

The Hutchinsonian notion of "ecological niche" is closely tied to the general understanding of "habitat". It is a multi-dimensional concept, that incorporates, environmental variation, biological constraints related to these variations implict in any ecosystem -- all relative to a given organism or species of interest. It is a highly context/organism-specific concept. 


\subsubsection{Habitat modeling -- species level}
\label{sec:habitatSpecies}
One frequently encountered means of describing this organism-specific habitat is to examine the presence or absence of an organism as a function of environmental and/or biological constraints. The probability of a given location to be potential habitat for a species of interest. The utility of such an approach is most relevant for sessile and slow moving organisms with highly specific habitat requirements. 

The simplest approach is to model the presence or absence as a Bernoulli process. In particular, the binomial or negative binomial representation of this process lends itself to a probability model that can be readily formulated as a simplistic binomial General Linear Model. However, as species-specific environmental constraints are generally modal in form, a nonlinear model is more useful. Generalized Additive Model (GAM; via the mgcv library) and a Stochastic Partial Differential Equation approach (SPDE/INLA; via the RINLA library) are two methods that can deal with these environmental constraints in a simple and efficient manner. They are used in this framework.

The intent is to develop simple predictive habitat models for wolffish, cod, and other species of interest to assess changes in the habitat area available. At present, only the models for snow crab have been completed (Figure~\ref{fig:snowcrabHabitat}; see \cite{snowcrabResDoc} for more details). It is possible to model the spatial/temporal patterns (Section~\ref{sec:interpolation}) and then integrate them in a risk-based approach (Section~\ref{sec:riskapproach}) to permit formal statements of risk and probability of exceeding thresholds. The likelihood of it being completed depends upon the time/resources available. No promises are being made here.

\begin{figure}[h]
  \label{fig:snowcrabHabitat}
  \centering
  \includegraphics[width=0.6\textwidth]{\ecomod snowcrab/R/gam/maps/{prediction.habitat.mean.R0.mass.2010}.png}
  \caption{Predicted probability of observing snow crab in Maritimes Region.}
\end{figure}


\subsubsection{Integrative habitat -- community level}

Over and above the rather simplistic species-habitat concept described in the previous section~\ref{sec:habitatSpecies}, there exists the more encompassing or integrative notion of habitat in the whole system context. Though the species-habitat concept itself does not exclude species interactions, in actual practice, they are generally ignored as they make the statistical estimation impossibly over-parameterized (but see snow crab assessment methods that makes it possible to incorporate some of them -- \cite{}). However, the result of these biological interactions are exactly what are reflected in the observed species assemblages. 

For example, two dominant species assemblages (communities) are known in the SSE, one that is related to depth gradients and another to temperature gradients. One can therefore examine solely these associated environmental forcing factors (temperature and depth) and determine the overall "habitat" space for the organisms extant in the system. However, this would ignore other factors that are either poorly known, poorly sampled, or poorly parameterized (e.g., dissolved oxygen, pH, redox, bacteria, jellyfish, squid, pollution, substrate type, etc.), factors that are theoretically important but difficult to demonstrate in practice. In contrast, the observed species composition would reflect all these and other unmeasured influences as they manifest the state of the system, that is, the full habitat to which these organisms are exposed.   

These associations are quantified using a standard multivariate procedure, namely Principal Components Analysis which focuses upon the correlational structure of the species assemblages (Figure~\ref{fig:speciesCompositionMap}) 

\begin{figure}[h]
  \label{fig:speciesCompostionMap}
  \centering
  %  \includegraphics[width=0.75\textwidth]{\ecomod speciescomposition/analysis/SSE/allseasons/complex/maps/{maps.pca1.2010}.png}
  \includegraphics[width=1.0\textwidth]{\mpa oneoffs/ca.png}
  \caption{Species composition variations in Maritimes Region.}
\end{figure}

The intent is to model the spatial/temporal patterns (Section~\ref{sec:interpolation}) and then integrate them in a risk-based approach (Section~\ref{sec:riskapproach}) to permit formal statements of risk and probability of exceeding thresholds. 



\subsection{Spatial and temporal statistical modeling and interpolation}
\label{sec:interpolation}
Source code: https://github.com/jae0/ecomod/spacetime/
	spde::spat.time
- mult/single what is the best approach?
- simultaneous covar/depend var estimation
- dynamic time series model method using the parameterization of Schnute?
- estimate spatially explicit timeseries autocorrelation for each main variable --- very do-able!
- relationship with corelation dim and autocorelation:: one is number (in state space) the other is variance (geographic space), but variance and number relationships are such that they can be related?
	stoch ts mod with habitat and bayes state space::


\subsection{Connectivity: space and time scales}

\subsubsection{Spatial scale}

The spatial scale of productivity, diversity and habitat will determine which processes will be relevant to an MPA. If the spatial variations in the productivity of a species of interest is small relative to the size of an MPA, the chances of the MPA having an influence upon the species is enhanced. This is usually the case when short-range processes dominate (e.g., less mobile species, weakly dispersing, low currents, hhabitat heterogeneity at small scales). If, however, the spatial scale is larger than the MPA, then it would mean that broader, regional processes were influencing the productivity of the species (e.g., higher mobility or dispersal processes/current, and stronger spatial connectivity, habitat heterogeneity at larger scales) -- resulting a lower likelihood of the MPA having an influence upon the species. 

For this rather simple reason, defining the spatial scale of a given observation or process is critical for the development of any assessment or monitoring of MPAs. To this end, we mush define what we mean by spatial scale. Any spatially defined observation can be seen as a realization of some stochastic spatial process. Such  processes are known as \textbf{spatial random fields} or \textbf{spatial stochastic process}, $y$, defined at some location $x$. The manner in which the variability of $y$ changes as a function of distance $\Delta x$ is very informative and is known as the \textbf{semivariogram} $\gamma$:

\begin{eqnarray*}
\label{eq:semivariogram}
\gamma(x) &=& \frac{1}{2} \ \textrm{Var} [ y(x) - y(x + \Delta x)]  \\
&=& \sigma^2 [1-c(x)] 
\end{eqnarray*}

where $\sigma^2$ is the total variance in observations $y$. It has a clear relationship to the spatial correlation function $c(x)$. 

Empirically, one generally observes that the variability is lower between locations closer together and higher for locations further apart. Eventually this variability levels off where observations at one location is completely disconnected from another. This spatial dependence or autocorrelation is important as it defines the spatial scales of a given process. 

Numerous models are used to parameterize this shape of this relationship. Well known ones include: spherical, exponential, Gaussian and Mat\'{e}rn models. In this report, we utilize the Mat\'{e}rn due to its flexibility and direct relationship to Stochastic Partial Differential Equations (SPDE's) which will become important later for interpolation purposes. 

\begin{equation}
\gamma(x) = \tau^2 + \sigma^2 [ 1 - \frac{{(2\sqrt{\nu}x)}^\nu}{2^{\nu-1} \rho^{\nu} \Gamma(\nu)}
 K_\nu(\frac{2\sqrt{\nu}x}{\rho} ) ]
\end{equation}

Here, $\tau^2$ is the variance offset often known as the nugget variance, the variability at local scales (usually associated with sampling error; $\nu$ controls smoothness/curviness; $\Gamma(\nu)$ is the Gamma function; and $K_{\nu}$ is the modified Bessel function of the second kind and order $\nu$. When $\nu=1/2$ then this become the exponential (or Laplacian) model. When $\nu \longrightarrow \infty$, the model becomes the Gaussian model. The spatial range can be recovered from this as the distance at which spatial correlation drops to some low level. In the RINLA estimation procedure (\cite{rinla}), this happens to be when correlation drops to $\approx$~0.13 which will also be used to define spatial range in this report. 


\begin{figure}
  \label{fig:matern}
  \centering
  \includegraphics[width=0.5\textwidth]{\ecomod mpa/matern.jpg}
  \caption{Matern semivariance as a function of distance for different values of $\nu$.}
\end{figure}

\subsubsection{Environmental data}

T

\subsubsection{Zooplankton and phytoplankon}

Zooplankton and phytoplankton standing biomass (and net production) will be examined to determine spatial and temporal ranges. The same will be attempted for satellite imagery data. The expectation is that the scales will be broad (larger than the size of SAB) and temporally short (annual to sub-decadal) due to a stronger influence of environmental/climatic scales of variation for these surface/planktonic processes. A risk-based model might be adopted depending upon availability of time and appropriateness of the space-time scales.

\subsubsection{Tagging, mark-recapture}
Increased tagging efforts have been made in the vicinity of SAB. Most of the effort has been driven by industry, Ocean Tracking Network, and Emera interest in snow crab movement near SAB (see Section~\ref{(se:snowcrab)}. However, acoustic tagging of other species of interest will be completed for cod, wolfish, halibut and others. This information will also help define the spatial connectivity and range of the species of interest. The intent is to develop movement models where possible and estimate spatial range directly.  

Mark-recapture information for sea turtles, seals, and various other species exist in the area. This data has not been examined. It is a data gap at present. 




\subsection{Risk modeling}
\label{sec:riskapproach}

The estimation of risk requires a reliable estimation of error. One way to propagate the errors from observations to predictions is to build either a deterministic model and then using well-understood approximations or simulations to ultimately determine the errors distributions of interest. A second method is more phenomenological, quantifying empirically the errors and propagating them via statistical/correlational methods. The latter is chosen in this framework as it is more general and simpler. There is known no operating model of an ecosystem that can be said to be able to perform with sufficient skill to be able to propagate errors, let alone, magnitudes. 

The proposed candidate approach is to use a the logistic equation, a well known approach that was made popular by Verhulst, Pearl and Lotka in the late 1800s and early 1900s and then again by the "Systems Theorists" in the 1960-1970s.  

- diversity: total, zooplankton, benthic invert, demersal fish?
- habitat space ..
- habitat defined as {``}habitat for species assemblage 1, 2, 3 (from classification methods? ) or
- simply habitat of a few charismatic species
- charismatic species: cod, wolffish, etc.
	spatially explicit range estimates:
- depth
- others:: substrate?, temperature? chl? production? nutrients? biota? -- time dependent


\subsubsection{Logistic model}

As observed by Lotka (1925), the rate of change in time $t$ of any state $Y$ (in our context: abundance, species richness, "habitat", size) can be expected to be in some minimal way, a function of itself:  

\begin{eqnarray} 
\label{eqLogisticContinuous}
dY / dt = g(Y)
\end{eqnarray}

It is expected that when $Y = 0$, $dY/dt$ will also be zero and so represents an algebraic root of $g$. A Taylor series expansion of $g$ near this root $Y=0$ gives:

\begin{eqnarray*} 
  \label{eqLogisticTaylorSeries}
  dY / dt &=&  g'(0) Y + g''(0) Y^{2}/2  + \text{higher order terms} \dots; \\
  &\approx& Y [g'(0) + g''(0) Y/2  ]
\end{eqnarray*}


With the identities $g'(0) = r$  and $g''(0) = - 2r/K$, the standard form of the logistic equation (also known as the Verhulst equation) is obtained:

\begin{equation} 
\label{eqLogistic}
dY/dt \approx r Y (1 - Y/K)
\end{equation}

The intrinsic rate of increase, $r$, is therefore, some abstract function f that regulates increase or decrease of the system state $Y$. In biological systems, this means growth, recruitment, mortality,  movement, climatic change, etc. The carrying capacity $K$ represents the upper limit of the magnitude of the system state $Y$.

With normalization by $K$ we obtain: $ y = Y/K $, which simplifies the basic model to:

\begin{equation} 
\label{eqLogisticBasic}
dy/dt  \approx  r K y (1 - y)
\end{equation}

Many variations of this basic model have been used, mostly to adjust the shape of the curve and/or the location of the inflection point (i.e., $K/2$) mostly by adding additional parameters in the form of exponentiation of different components which amounts to adding higher order terms in the Taylor series) and also adding additional terms, $f$ (e.g., fishing, advection, diffusion, noise): 

\begin{eqnarray} 
\label{eqLogisticGeneral}
dy / dt &=& rK^{\beta-\alpha} y^{\alpha} ( 1 - y^{\beta} )^{\gamma} + f 
\end{eqnarray}

For the purposes of this discussion, we will use only the basic model and estimate the parameters of interest, $\theta = \{r, K \}$ (Eq. \ref{eqLogisticBasic}), but the reader should be aware that other parameterizations are possible such as Eq. \ref{eqLogisticGeneral}, at the expense of course, a slightly greater model complexity $\theta=\{r,K,\alpha,\beta,\gamma\}$. The utility of these other formulations will be explored once all data assimilation has been completed. 

In discrete form, where $\Delta t = 1 \; \text{year}$, the basic model becomes:

\begin{equation} 
\label{eqLogisticDiscrete}
y_t  \approx y_{t-1} + r K y_{t-1} (1 - y_{t-1} )
\end{equation}

Perhaps the most powerful and flexible method of estimation of $\theta$ in (Eq. \ref{eqLogisticDiscrete}) is a state space representation where an additional observation model is added to connect observed indices $O$ to the unobserved and (usually unobservable) real system state $y$. The simplest such model is a simple linear scaling factor, $q$: 

\begin{equation} 
\label{eqLogisticDiscreteStateSpace}
O_t = q y_t
\end{equation}


The use of a Bayesian approach to solve the above nonlinear state space problem (Eqs. \ref{eqLogisticDiscrete}, \ref{eqLogisticDiscreteStateSpace} ) is frequently used as it provides an opportunity to: have greater numerical stability;  incorporate prior scientific knowledge in a formal manner; realistically propagate credible errors; estimate unobserved states ("true" fishable biomass); and simultaneously estimate model "process" errors and data "observation" errors. 

Process errors ($\sigma^2_p$) are the uncertainties that feeds back into future states via error propagation -- e.g., via the recursive form of the logistic equation (i.e., errors in $y_{t+1}$ in the state space of $y_t$ vs $y_{t+1}$). Observation errors ($\sigma^2_o$) refer to the uncertainties associated with measurement and observation (i.e., measurement/data-related errors of both variables in the state space of $y_t$ vs $y_{t+1}$). This latter ability is particularly important as parameter estimates and forecasts based on observation-only errors provide unrealistically optimistic (small and constant) error bounds; and parameter estimates and forecasts based on process-only errors expand rapidly into the future, resulting in potentially unrealistically pessimistic (large and usually growing) error bounds.
%
%The main distributional assumptions of the model of fishable biomass are as follows. The notation "$\sim$" indicates "distributed as". The elements in the parentheses are (mean, variance), or in the case of the uniform distribution (lower interval, upper interval).  
%
%\begin{eqnarray*} 
%\label{eqPriors}
%  q &\sim& \text{Uniform} ( 0.1, 2 )  \\
%  K  &\sim&  \text{Lognormal} ( \mu_K, \sigma^2_K ) \\
%  r  &\sim&  \text{Normal} (\mu=0.96, \sigma^2=0.01) \\
%  O_t  &\sim&  \text{Lognormal} ( log(q K y_t), \sigma^2_o ) \\
%  y_t  &\sim&  \text{Lognormal} ( log(y_{t-1} + r_{t-1} y_{t-1} (1 - y_{t-1} ) ), \sigma^2_p ) \\
%\end{eqnarray*}
%
%Most ecological variables follow a lognormal error distribution. A multiplicative observation error model was assumed. The scaling factor $q$, simplistically quantiﬁes the inﬂuence of a number of differing biases, including survey gear, survey protocols, areal expansion protocols, survey stratification and statistical modeling, etc. As such biases are non-constant over time and space it is hopelessly inadequate; however, it is retained for simplicity as a first order of magnitude estimate of such influences. The prior of q was assumed to be uniform in distribution, bounded by the interval (0.1, 2). Process error was assumed to follow a (multiplicative) lognormal distribution with variance $\sigma^2_p$ whose prior was lognormally distributed. Carrying capacity was assumed to follow a lognormal distribution. Where the area specific $\mu_K$ and $\sigma^2_K$ were chosen based on previous knowledge of the production in the area. The intrinsic rate of increase was assumed to be stationary with a prior of.
%These priors were marginally informative. 

The posterior distribution of the parameters of interest, $\theta$, conditional upon the data were estimated via MCMC (Gibbs) sampling using the JAGS platform (Plummer 2003, 2010). The JAGS model used for parameter estimation can be found at: \\

https://github.com/jae0/ecomod/mpa/bugs/




\section{Results}

NA

\section{Discussion}

NA

\section{Conclusions and recommendations}

NA



%%%% bibliography %%%%%

\clearpage

%\bibliographystyle{unsrt}
%\bibliography{sab.bib}

% \printbibliography


\clearpage \newpage


\begin{appendices}
  
\section{Data quality control}

All data extraction, quality control and processing methods are documented in R scripts found in https://github.com/jae0/ecomod/.

\subsection{Discrete bottle data}

BioChem data are generally not subject to any quality control (QC). As such, substantial QC was required. The QC protocol used was based on procedures designed at DFO's Institut Maurice-Lamontagne (IML). These were in turn was based on procedures developed by NOAA's National Oceanographic Data Center/World Ocean Database (Conkright \textit{et al}. 2002) as well as many of the tests proposed in the GTSPP Real-Time Quality Control Manual (Unesco 1990). The quality control procedure was as follows:

\textbf{Step 1: Impossible dates}

Due to known issues with dates, database query for nutrients and chlorophyll were designed to extract records for which the sampling date is within start and end dates of the mission. Another check includes comparing HEADER\_START and EVENT\_START dates which should be the same. It is often found that month and day were reversed in EVENT\_START field. Those records were retained and HEADER\_START was used as a more reliable date.

\textbf{Step 2: Quality control flags}

Small number of records in BioChem was subject to quality control and  include flags for position (POSITION\_QC\_CODE) and data (DATA\_QC\_CODE). The meaning of the codes are as follows:

\begin{table} [h]
\begin{tabular}{ll}
0 & No quality control performed \\
1 & Value appears correct \\
2 & Value appears inconsistent \\
3 & Value appears doubtful \\
4 & Value  appears erroneous \\
5 & Value changed as result of quality control \\
\end{tabular}
\end{table}

QC flags were checked for parameter and inconsistent (2), doubtful (3) and erroneous (4) values were removed from the dataset.

\textbf{Step 3: Depth check}

For bottle data, start and end depth at which the water samples were collected are verified to be the same. Records with different start and end depths were removed from the dataset.

\textbf{Step 4: Duplicated records}

BioChem often contains duplicated records as same data was sometimes loaded into database twice and treated as different records. Duplicated records were removed and the first record of each duplicate is kept in the dataset.

\textbf{Step 4: Suspect missions}

Missions with suspect data were identified and removed. Those missions often show unusual data values (for example, integer numbers without decimal places with only some values out of range) suggesting that the data for the whole mission might be compromised. The suspect mission for each parameter are following:

\begin{table} [h]
  \begin{tabular}{ll}
	Chlorophyll-a & OC7908, 32G879008 \\
  Phosphate & 18HU88026 \\
  Silicate & 180167005, 31TR26870 \\
  \end{tabular}
\end{table}


\textbf{Step 5: Global impossible parameter values}

Chlorophyll and all nutrients values were examined if they fell within expected limits for NW Atlantic, which are adopted from IML quality control procedure (IML Test 2.1). The expected range of values are:
\begin{itemize}
  \item Chlorophyll-a: 0-50 mg/m\textsuperscript{3}
  \item Nitrate: 0-515 mmol/m\textsuperscript{3}
  \item Phosphate: 0-4.5 mmol/m\textsuperscript{3}
  \item Silicate: 0-250 mmol/m\textsuperscript{3}
\end{itemize}

Any values outside of expected range were removed for open ocean data only. Coastal data ( up to 5km from the coast) were not filtered using expected ranges as in coastal water chlorophyll-a and nutrients concentrations can be higher. 

\textbf{Step 6: Profile envelope}

Data  for each parameter are checked if they fall within the expected limits by depth interval, as shown in Table~\ref{table:BioChemQC} (IML Test 2.4). This test does not allow zero values for silicate and phosphate in the deep water. Again, only open ocean data were subject to the profile envelope test.

\begin{table}[h]
\label{table:BioChemQC}
\caption{Expected ranges of parameters for the profile envelope test (IML Test 2.4)}
\begin{tabular}{lll}
Parameter & Depth interval & Expected Range \\
\hline
Chlorophyll-a & 0-1500 m & 0-50 mg/m$^{3}$ \\
Silicate & 0-150 m & 0-250 mmol/m$^{3}$ \\
Silicate & 150-900 m & 0.01-250 mmol/m$^{3}$ \\
Phosphate & 0-500 m & 0-4.5 mmol/m$^{3}$ \\
Phosphate & 150-1500 m & 0.01-4.5 mmol/m$^{3}$ \\
Nitrate & 0-1500 m & 0-515 mmol/m$^{3}$ \\
\end{tabular}
\end{table}


\textbf{Step 7: Impossible profiles}

This check was not implemented in the code and impossible profiles were identified by investigating unusual outliers.

Additional steps from IML QC procedure such as checks for constant profile, excessive gradient and inversions were not implemented in this quality control procedure. However, due to eutrophication from terrestrial sources, phosphate levels in coastal regions often exceeded the upper limits for globally and locally possible values, with the phosphate concentrations sometimes six times higher than the upper limits for the NW Atlantic in offshore waters. Therefore coastal and open ocean data were examined separately, non-coastal data were filtered using the expected limits for the NW Atlantic. Coastal data were defined as the ones collected less than 5 km away from the coast, where 5 km limit was chosen as an optimal distance at which all coastal inlets are included. Buffer polygons along the coastline were created (Figure~\ref{fig:aoiBuffer}) and used for flagging the data as open ocean records (flag 1) and coastal records (flag 2).

\begin{figure}
  \label{fig:aoiBuffer}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 26.png}
  \caption{Polygons used for separation of coastal and ocean data, arbitrarily assumed to be a 5 km buffer from the coastline.}
\end{figure}

\clearpage

\subsection{Chlorophyll-a}

Chlorophyll-a data are derived from four methods. The methods are listed and described in Table~\ref{table:ChlaMethods} and the aggregate time series  associated with each method is shown in Figure~\ref{fig:ChlaTimeseries}. For most of the chlorophyll data the method is not specified (unknown); Holm-Hansen fluorometric method is the standard AZMP method and is the second most frequent; Welschmeyer fluorometric method is used least frequently, often by the Quebec and Newfoundland regions.

In a number of cases, the same water sample was processed using two different methods, resulting in two sets of chlorophyll estimates for the same samples. Comparisons between these two sets of values are shown in Figure~\ref{fig:ChlaComparison}. In both cases chlorophyll-a estimated by the Welschmeyer method are lower than the ones using the Holm-Hansen method or "unknown" method. Since there is more data mapped to the Holm-Hansen method than to the Welschmeyer method, only data derived from Chl\_a and Chl-a\_Holm\_Hansen\_sF methods were retained. No corrections were applied to correct for differences in methodology.


\begin{table}[h]
  \label{table:ChlaMethods}
  \caption{Methods associated with chlorophyll-a records in BioChem.}
  \begin{tabular}{ll}
    Method & Description \\ \hline
    Chl\_a & Unknown method \\
    Chl\_a\_Holm-Hansen\_F & Holm-Hansen method; Prefiltered; Frozen before analysis (-20 \textcelsius )  \\
    Chl\_a\_Holm-Hansen\_sF & Holm-Hansen method ; Super Frozen before analysis (-196 \textcelsius) \\
    Chl\_a\_Welschmeyer\_sF & Welschmeyer method; Super Frozen before analysis  (-196 \textcelsius) \\
  \end{tabular}
\end{table}


\begin{figure}[h]
  \label{fig:ChlaTimeseries}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 27.png}
  \caption{Time series of chlorophyll-a data from BioChem grouped by methods.}
\end{figure}


\begin{figure}[h]
  \label{fig:ChlaComparison}
  \centering
  \begin{tabular}{cc}
    \includegraphics[width=0.5\textwidth]{\sab 28.png}
    \includegraphics[width=0.5\textwidth]{\sab 29.png} 
  \end{tabular}
  \begin{tabular}{cc}
    \includegraphics[width=0.5\textwidth]{\sab 30.png}
    \includegraphics[width=0.5\textwidth]{\sab 31.png}
  \end{tabular}
  \caption{Comparison of the chlorophyll-a values collected using different methods is shown on the scatter plots on the top panel and the geographical locations of those samples is shown on the maps in the bottom panel.}
\end{figure}



\clearpage

\subsection{Nitrate}

% Table begins
% 
\begin{table}[h]
  \label{table:Nmethods}
  \caption{Methods associated with nitrate records in BioChem.}
  \begin{tabular}{ll}
    Method & Description\\
    \hline
    NO2NO3\_0 & Nitrate+Nitrite / Unknown method \\
    NO2NO3\_Alp\_F & Nitrate+Nitrite / Alpchem / Frozen \\
    NO2NO3\_Alp\_SF & Nitrate+Nitrite / Alpchem / SuperFrozen \\
    NO2NO3\_S\&P1968 & Nitrate+Nitrite/ S\&P(1968) / filtered and frozen \\
    NO2NO3\_Tech\_F & Nitrate+Nitrite / Technicon / Frozen \\
    NO2NO3\_Tech\_Fsh & Nitrate + Nitrite / Technicon / Fresh / Strain / Unfiltered \\
    NO2NO3\_Tech\_SF & Nitrate+Nitrite / Technicon / SuperFrozen \\
    NO2NO3\_Tech2\_F & Nitrate+Nitrite / Technicon2 / Frozen \\
    NO3\_Tech\_F & Nitrate / Technicon / Frozen, corrected for NO2 \\
    NO3\_Tech\_SF & Nitrate / Technicon / SuperFrozen\\
  \end{tabular}
\end{table}

\begin{figure}[h]
  \label{fig:NTimeseries}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 32.png}
  \caption{Time series of nitrate data from BioChem grouped by methods.}
\end{figure}

Nitrate estimates are derived from 10 methods. The methods are listed and described in Table~\ref{table:Nmethods} and the time series of data associated with each method is shown in Figure~\ref{fig:NTimeseries}. Most of the methods measure nitrate and nitrite together. We also included data for nitrate only since  in most seawater the concentration of nitrite is small compared to that of nitrate.

\clearpage
\subsection{Phosphate}

Phosphate  data available in BioChem are mapped to 7 methods. The methods are listed and described in Table~\ref{table:PhosphateMethods} and the time series of data associated with each method is shown in Figure~\ref{fig:PhosphateTimeseries}.

\begin{table}[h]
  \caption{Methods associated with phosphate records in BioChem.}
  \label{table:PhosphateMethods}
  \begin{tabular}{ll}
    Method & Description \\ \hline
    PO4\_0 & Phosphate / Unknown method  \\ 
    PO4\_Alp\_SF & Phosphate / Alpchem / SuperFrozen / Filtered \\ 
    PO4\_Tech\_2 & Phosphate / Murphy and Riley / filtered and frozen \\ 
    PO4\_Tech\_F & Phosphate / Technicon / Frozen / Unfiltered \\ 
    PO4\_Tech\_Fsh & Phosphate / Technicon / Fresh / Strain / Unfiltered \\
    PO4\_Tech\_SF & Phosphate / Technicon / SuperFrozen / Filtered \\
    PO4\_Tech2\_F & Phosphate / Technicon2 / Frozen / Unfiltered \\
  \end{tabular}
\end{table}


\begin{figure}[h]
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 33.png}
  \caption{Time series of phosphate data from BioChem grouped by methods.}
  \label{fig:PhosphateTimeseries}
\end{figure}


\clearpage
\subsection{Silicate}

Silicate data available in BioChem are mapped to 8 methods (Table~\ref{table:SilicateMethods}). Their time series of are shown on Figure~\ref{fig:silicateTimeseries}.

\begin{table}[h]
  \label{table:SilicateMethods}
  \caption{Methods associated with silicate records in BioChem.}
  \begin{tabular}{ll}
    Method & Description \\ \hline
    SiO4\_0 & Silicate, Unknown methods and handling \\
    SiO4\_1 & Silicate / Mullin and Riley / filtered and frozen \\
    SiO4\_Alp\_F & Silicate / Alpchem / Frozen / Unfiltered \\
    SiO4\_Alp\_SF & Silicate / Alpchem / SuperFrozen / Filtered \\
    SiO4\_Tech\_F & Silicate / Technicon / Frozen / Strain / Unfiltered \\
    SiO4\_Tech\_Fsh & Silicate / Technicon /Fresh /Strain / Unfiltered \\
    SiO4\_Tech\_SF & Silicate / Technicon / SuperFrozen / Filtered \\
    SiO4\_Tech2\_F & Silicate / Technicon2 / Frozen \\
  \end{tabular}
\end{table}


\begin{figure}[h]
  \label{fig:silicateTimeseries}
  \centering
  \includegraphics[width=1.0\textwidth]{\sab 34.png}
  \caption{Time series of silicate data from BioChem grouped by methods.}
\end{figure}

\clearpage
\subsection{Zooplankton}

Zooplankton data was extracted from DFO's BioChem database (DFO 2015, Devine \textit{et al}. 2014) from 1914 to 2014. They were comprised of 687 missions and 53,787 samples, using 13 different kind of nets, 35 different mesh sizes ranging from 20 microns to 4.23 mm, and various net deployment and sample processing protocols.
To ensure data consistency and comparability, only samples collected and analyzed using Atlantic Zone Monitoring Program (AZMP) protocol (Mitchell \textit{et al}. 2002) were retained for study, reducing the time scope to 1999-2014. 

As AZMP samples are not always properly flagged in the BioChem database, a list of missions that followed the AZMP protocol were provided by Ocean and Ecosystem Science Division (OESD) and Ocean Data and Information Services (ODIS). The relevant missions include AZMP spring and fall cruises, summer and winter groundfish survey missions, bi-weekly sampling at fixed stations (Halifax Station 2 and Prince 5) and samples collected on the Scotian Shelf during Labrador Sea missions. 

AZMP protocol samples zooplankton with Ring nets (0.75 m diameter,mesh size of 202 um) deployed as vertical tows from either near bottom or 1000 m (whichever is shallower) to the surface. The sample analysis includes estimation of abundance, species composition and biomass in terms of wet and dry weight in two size fractions; one for organisms ranging from 0.2 mm to 10 mm in size and the other for all organisms larger than 10 mm.  The protocol is as follows:

\begin{itemize}
  \item Organisms larger than 10 mm are manually separated from the sample, identified and counted. Wet weight is determined and reported for each individual species. In addition total wet weight for all large organisms is reported as the sum of all individual wet weights.
  \item Captured organisms smaller than 10 mm (0.2 - 10 mm fraction), are identified and counted. Dry and wet weight is determined and reported for the whole sample, containing all organisms in that size fraction.
  \item Total wet weight is reported for all captured organisms as the sum of wet weights of large and small organisms.
  \item Developmental stages are identified for \textit{Calanus finmarchicus}, \textit{Calanus glacialis} and \textit{Calanus hyperboreus}. 
\end{itemize}


Since the data hosted in BioChem is generally not subject to quality control and can contain erroneous records substantial quality control was conducted to ensure correct representation of actual measurements. The quality control included verification of the following fields: 

\begin{itemize}
  \item time stamps: comparing mission dates with header dates and event dates 
  start and end depths of the nets that cannot be equal or close together
  \item volume of the samples: all records with volumes 0, or NA were removed
  \item split fraction of the sample: all the records with split fraction NA, 0 or $>$ 1 were removed 
  \item minimum and maximum sieve for dry weight records: records NA sieve were removed
  \item repeated records 
\end{itemize}


Finally the numerical and biomass density for each species per unit surface area of a tow was computed as follows: 
\begin{eqnarray*}
\text{abundance} &=& \text{counts} \times \frac{  abs( \text{depth}_{\text{start}} - \text{depth}_{\text{end}} ) } { \text{split fraction} \times \text{volume} } \; \; [\text{individuals}/ \text{m}^{2} ]  \\
\text{biomass} &=& \text{weight} \times \frac{ abs( \text{depth}_{\text{start}} - \text{depth}_{\text{end}} )} { \text{split fraction} \times \text{volume}} \; \; [\text{g}/\text{m}^2]
\end{eqnarray*}

where counts refer to number of organisms encountered in the sample, $\text{depth}_{\text{start}}$ and $\text{depth}_{\text{end}}$ to the start and end depth of the net deployment, split fraction to the fraction of the sample analyzed and volume to the sample volume. The final filtered dataset includes 126 missions in the time period 1999 to 2014, with 2,367 net deployments and more than 400 taxonomic species. 

\end{appendices}

\end{document}
